src/apps/profiles/models.py

```diff
@@ -0,0 +1,16 @@
+from django.db import models
+from django_extensions.db.models import TimeStampedModel
+from django.contrib.auth import get_user_model
+from phonenumber_field.modelfields import PhoneNumberField
+
+User = get_user_model()
+
+
+class Profile(TimeStampedModel):
```

Comment:
can we add a specialist field to differentiate a user from a doctor
```suggestion
class Profile(TimeStampedModel):
     is_specialist = models.BooleanField(default=False)
```

Reply:
Cool will update